WordPress/Classes/Plugins/Views/InstalledPluginsListView.swift

```diff
@@ -54,6 +54,9 @@ struct InstalledPluginsListView: View {
                 AddNewPluginView(service: viewModel.service)
             }
         }
+        .onApplicationPasswordUpdate {
```

Comment:
(nit) I think it's an over-kill to add a ViewModifier just to listen to a notification.

Reply:
I agree. An extension function is enough here.